hashicorp/boundary-ui
# refactor: 💡 merge account mirage factories

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-12888

## Description

merge account mirage factories and update account id to be type specific



## Screenshots (if appropriate):

## How to Test

You can utilize the vercel deployment for admin UI. You can visit the `/users/u_1r8fkwwiqj/add-accounts` route to see a list of all the accounts mirage generated. You should now see `acctpw_`, `acctldap_`, or `acctoidc_` for the respective account types.





:technologist: [Admin preview](https://boundary-ui-git-icu-12888-account-hashicorp.vercel.app/scopes/global/users/u_1r8fkwwiqj/add-accounts)

## Checklist:
